.github/scripts/check-header.py

```diff
@@ -105,76 +113,352 @@ def configArgParser():
                         help    = 'Treat arguments json files that store a list of files to check.')
     return parser
 
+#--------------------------------------------------------------------------------------------------
+#                                            CONFIG  
+#--------------------------------------------------------------------------------------------------
+FREERTOS_IGNORED_EXTENSIONS = [
+    '.1',
+    '.ASM',
+    '.C',
```

Comment:
Is this right? I think cpp files should have the same header

Reply:
It's correct for current state of repo in that no `FreeRTOS/FreeRTOS` files of these types include the header. These should be skipped. 

The changes in `header-checks.yml` now use the check as defined in the user PR so users can update as needed